src/bp/core/api.ts

```diff
@@ -133,6 +133,12 @@ const users = (userRepo: UserRepository): typeof sdk.users => {
 
 const kvs = (kvs: KeyValueStore): typeof sdk.kvs => {
   return {
+    forBot(botId: string): KvsService {
+      return new KvsService(kvs, botId)
+    },
+    global(): KvsService {
+      return new KvsService(kvs)
```

Comment:
Creating a new instance for every read and write to the kvs is gonna be kind of heavy, can you implement this similarly to how the ghost was done ? The forBot stores the scoped version in a cache and returns it in future calls.  (I just saw that global isn't doing it - will fix that)